composer.json

```diff
@@ -20,20 +20,22 @@
             "homepage": "https://github.com/friendsofsymfony/FOSHttpCache/contributors"
         }
     ],
+    "minimum-stability": "dev",
```

Comment:
@andrerom Definitely. FOSHttpCache 2.0 will not ship before we can remove the `minimum-stability` flag.